jobs/keepalived/spec

```diff
@@ -20,8 +20,8 @@ properties:
     description: when health check fails, this triggers a fail over. The default command checks the haproxy process is still alive.
     default: killall -0 haproxy
   keepalived.interface:
-    description: interface keepalived will use to mount the VIP
-    default: eth0
+    description: interface keepalived will use to mount the VIP. If set to 'auto', uses the default interface on the VM
```

Comment:
Seems the proposed change identifies the default interface as the one routing the "8.8.8.8" IP address. Would a wording such as the following more explicit ?

``description: interface keepalived will use to mount the VIP. If set to 'auto', uses the default interface on the VM (i.e. the one used to route egress internet traffic to "8.8.8.8"), which might be a convenient default for use in bosh-lite.``